server/routes/user.routes.js

```diff
@@ -430,4 +430,25 @@ router.post(
   UserController.removePropertyFromFavorites,
 );
 
+/**
+ * @swagger
+ * path:
+ *  /user/overview:
```

Comment:
Overview is too generic, lets make this to be account-overview so that it can be explicit.
```suggestion
 *  /user/account-overview:
```